Geometry/TrackerNumberingBuilder/plugins/CmsTrackerLevelBuilder.h

```diff
@@ -26,18 +22,38 @@ class CmsTrackerLevelBuilder : public CmsTrackerAbstractConstruction {
   static double getPhiGluedModuleMirror(const GeometricDet* a);
   static bool isLessRModule(const GeometricDet* a, const GeometricDet* b);
   static bool isLessR(const GeometricDet* a, const GeometricDet* b);
+};
 
-  void build(DDFilteredView&, GeometricDet*, std::string) override;
+/**
+ * Abstract Class to construct a Level in the hierarchy
+ */
+template <class T>
```

Comment:
can one use a name a bit more descriptive than T? (FilteredView for instance)?

Reply:
sure :-)

Reply:
done